### PR TITLE
Feature/sc 8599 remodel help area contact page

### PR DESCRIPTION
**Pull request by pauljanns**
# Description
I remodeled the help area contact page. The overall design changed and a dropdown with an iframe was added. The iframe probably won't load locally as it is with all confluence pages embedded as an iframe on test and staging. However, the URL can be checked manually to see how the iframe would look like. 

## Links to Tickets or other pull requests
- https://ticketsystem.schul-cloud.org/browse/SC-8599

## Changes
- Help Area contact page
- Dropdown with problem / wish area
- Iframe with known problems
- design changes

## Screenshots of UI changes
**### Old version -> notice two sections left and right**

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by luizzaly** [comment]
Can you please a screenshot of UI changes?

**Comment by luizzaly** [comment]
can you please add a screenshot of the UI changes

File: locales/en.json [hunk 1]
```diff
@@ -1798,7 +1798,43 @@
 			},
 			"text": {
 				"thankYouForYourFeedback": "Thank you for your feedback, it will be sent directly to the {{shortTitle}} development team. Please be careful not to send any passwords or other sensitive data.",
-				"whatKindOfFeedbackDoYouWant": "What kind of feedback would you like to send us?"			}
+				"whatKindOfFeedbackDoYouWant": "What kind of feedback would you like to send us?",
+				"whoDoYouWantToContact": "Who do you want to contact?",
+				"whichAreaOfSchulCloud": "Which area of Schul-Cloud does your topic concern?",
+				"contactSupport": "Contact support",
+				"contactSupportSubtitle": "Your problem isn't listed? Contact our support team.",
+				"contactSupportSubtitleWish": "You want to share a wish? Contact our support team."
```
Comment (luizzaly) [inline locales/en.json:1806]: Do you want to share a request? Please contact our support team.

File: locales/en.json [hunk 2]
```diff
@@ -1809,10 +1845,14 @@
 			"label": {
 				"maxSizeFiles": "Images, videos and files",
 				"problemDescription": "Problem description:",
-				"theProblemOccursOnTheDevice": "The problem occurs on this device"
+				"theProblemOccursOnTheDevice": "The problem occurs on this device",
+				"problemAlreadyKnown": "Is the problem already known?",
+				"knownProblemLink": "Here you can see if there are currently problems with the technical systems: ",
```
Comment (luizzaly) [inline locales/en.json:1850]: Here you can check if there are currently issues regarding the technical systems:

**Review by luizzaly — APPROVED** [review]

File: locales/de.json [hunk 3]
```diff
@@ -1798,7 +1798,43 @@
 			},
 			"text": {
 				"thankYouForYourFeedback": "Danke für dein Feedback, es wird direkt an das Entwicklungsteam der {{shortTitle}} gesendet. Bitte achte darauf, keine Passwörter und andere sensible Daten mitzusenden.",
-				"whatKindOfFeedbackDoYouWant": "Welche Art von Feedback möchtest du uns senden?"			}
```
Comment (Ojself) [inline locales/de.json:1801]: Is that curle bracket } misplaced?

**Review by Ojself — APPROVED** [review]

File: locales/de.json (hunk 3 above)
Comment (pauljanns) [reply locales/de.json:1801]: No, that's a linting thing.

**Pull request merged**